metapi/metapi.js

```diff
@@ -251,47 +251,94 @@ window.metapi = {};
    */
   metapi.getPost = function (ident, key, options, callback) {
     options = options || {};
+    options.key = key;
+    options.page = options.page || 1;
 
-    var overwrite = options.hasOwnProperty("forceReload") && delete options["forceReload"];
+    var overwrite = options.hasOwnProperty("forceReload") && options["forceReload"];
+    delete options["forceReload"];
```

Comment:
Why not tack this onto the end of the previous condition?

    options.hasOwnProperty("forceReload") && options["forceReload"]  && delete options["forceReload"];